src/modules/mc_pos_control/MulticopterPositionControl.cpp

```diff
@@ -269,8 +271,12 @@ void MulticopterPositionControl::Run()
 				}
 
 				failsafe(time_stamp_now, setpoint, _states, !was_in_failsafe);
+
 				_control.setInputSetpoint(setpoint);
+
 				constraints = {0, NAN, NAN, NAN, NAN, NAN, NAN, NAN, false, {}};
+				_control.setConstraints(constraints);
```

Comment:
Makes sense. At no point in time were constraints reset for the emergency failsafe. This eliminates the fault vector of malicious e.g. too narrow constraints being used when the setpoints are not executable 👍